src/core/connection.c

```diff
@@ -6641,6 +6657,25 @@ QuicConnParamSet(
         break;
     }
 
+    case QUIC_PARAM_CONN_QTIP: {
```

Comment:
Instead of making a param for this, I think it'd be better to indicate the `UseQTIP` state in the `QUIC_NEW_CONNECTION_INFO` struct that's indicated in the NEW_CONNECTION event to listeners.

Reply:
I am removing this in the next few iterations. We are fully migrating off of params to settings. I wanted to get the settings stuff working before deleting code.